packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol

```diff
@@ -10,6 +10,17 @@ contract HyperbridgeAdapter is BlockHashAdapter, Ownable, BaseIsmpModule {
     mapping(bytes32 => bytes32) public enabledReporters;
     mapping(bytes32 => uint256) public chainIds;
 
+    // @dev The address of the IsmpHost on the current chain
+    address private _host;
```

Comment:
i would make it `public` and i would remove `function host() public view override returns (address)`

Reply:
Unfortunately this method is required by the `BaseIsmpModule`